File: app/src/main/java/com/maximus/productivityappfinalproject/domain/ResetAppUseTimeDbUseCase.java
package com.maximus.productivityappfinalproject.domain;

import com.maximus.productivityappfinalproject.data.AppsRepository;

public class ResetAppUseTimeDbUseCase {

    private AppsRepository mAppsRepository;

    public ResetAppUseTimeDbUseCase(AppsRepository appsRepository) {
        mAppsRepository = appsRepository;
    }

    public void resetHourAppUsage() {
        mAppsRepository.resetHourly();
    }

    public void resetDayAppUsage() {
        mAppsRepository.resetDaily();
    }
}

File: app/src/main/java/com/maximus/productivityappfinalproject/domain/ResetAppUseTimeDbUseCase.kt
package com.maximus.productivityappfinalproject.domain

import com.maximus.productivityappfinalproject.data.AppsRepository
import javax.inject.Inject

class ResetAppUseTimeDbUseCase @Inject constructor(private var appsRepository: AppsRepository){

    fun resetHourAppUsage() {
        appsRepository.resetHourly()
    }

    fun resetDayAppUsage() {
        appsRepository.resetDaily()
    }
}